pkg/client/cache/eventqueue.go

```diff
@@ -400,6 +397,7 @@ func (eq *EventQueue) Resync() error {
 	for _, id := range eq.store.ListKeys() {
 		if !inQueue.Has(id) {
 			eq.queue = append(eq.queue, id)
+			eq.events[id] = watch.Modified
```

Comment:
That's a good point.  We should just say that if they are in the store, but not in the queue, they must be modified (since if the event was created or deleted, there should already be a queue entry)